composer.json

```diff
@@ -14,7 +14,7 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "phpunit/phpunit": "^4.0 || ^5.0",
+        "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0",
```

Comment:
This will not work with the current version of `zend-test`, because in PHPUnit 6 the class `[PHPUnit_Framework_TestCase ` no longer exists](https://github.com/sebastianbergmann/phpunit/blob/6.0/ChangeLog-6.0.md#removed).